### DISCUSSION TITLE
Responses from Copilot chat are cutting off

### DISCUSSION THREAD

**Discussion by robertcosborne** [community/community#62621] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When we ask Copilot in Visual Studio to convert a longer block of code, the response just cuts off at some point with no other output or messages. Asking it to do more doesn't change the result.

I found this issue was fixed in VS Code, but apparently not in the VS version?
https://github.com/microsoft/vscode-copilot-release/issues/303

**Comment by Akash1134** [comment]
Hi @robertcosborne ,

Are you using the latest version if Visual Studio and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.VSGitHubCopilot) extension?

The best place to submit Copilot Chat related feedback for Visual Studio is to use the in-product flow.

- Launch a developer prompt using any of the options mentioned here - https://learn.microsoft.com/en-us/visualstudio/ide/reference/command-prompt-powershell?view=vs-2022
- Report a problem from VS2022: https://aka.ms/vs/report-a-problem
- Run the command devenv /log (this will launch Visual Studio with verbose logging)
- Use the report a problem function to create a ticket
- Copy all the contents of the Github Copilot Output Window and add them to the ticket filed.

Thanks!
